src/bot/cogs/levels.py

```
@@ -0,0 +1,272 @@
+        current_time = time.time()
+        cooldown_key = (guild_id, user_id)
+
+        if self.cooldowns.get(cooldown_key, 0) > current_time - 60:
+             return
+        self.cooldowns[cooldown_key] = current_time
+
+        data = await self.bot.api.get_user_level(guild_id, user_id)
+        current_lv = data['level'] if data else 1
+        current_xp = data['xp'] if data else 0
+        
+        xp_to_add = random.randint(15, 25)
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)